publisher/publish.sh

```diff
@@ -132,18 +131,18 @@ function zipOntologyFiles () {
     # Make sure that everything is world readable before we zip it
     #
     chmod -R g+r,o+r .
-    zip -r ${zipttlDevFile} "${family_product_branch_tag}" -x \*.rdf \*.zip  \*.jsonld \*AboutFIBOProd.ttl
-    zip -r ${ziprdfDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.jsonld \*AboutFIBOProd.rdf
-    zip -r ${zipjsonldDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.rdf \*AboutFIBOProd.jsonld
+    zip -r ${zipttlDevFile} "${family_product_branch_tag}" -x \*.rdf \*.zip  \*.jsonld \*About${ONTPUB_FAMILY}Prod.ttl
+    zip -r ${ziprdfDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.jsonld \*About${ONTPUB_FAMILY}Prod.rdf
+    zip -r ${zipjsonldDevFile} "${family_product_branch_tag}" -x \*.ttl \*.zip \*.rdf \*About${ONTPUB_FAMILY}Prod.jsonld
 
     ${GREP} -r 'utl-av[:;.]Release' "${family_product_branch_tag}" | ${GREP} -F ".ttl" | ${SED} 's/:.*$//' | xargs zip -r ${zipttlProdFile}
-    ${FIND}  "${family_product_branch_tag}" -name '*About*.ttl' -print | ${GREP} -v "AboutFIBODev.ttl" |  xargs zip ${zipttlProdFile}
+    ${FIND}  "${family_product_branch_tag}" -name '*About*.ttl' -print | ${GREP} -v "About${ONTPUB_FAMILY}Dev.ttl" |  xargs zip ${zipttlProdFile}
```

Comment:
Should be uppercase:
`${FIND} "${family_product_branch_tag}" -name '*About*.ttl' -print | ${GREP} -v "About${ONTPUB_FAMILY^^}Dev.ttl" | xargs zip ${zipttlProdFile}`